src/wallet/rpcwallet.cpp

```diff
@@ -135,14 +135,15 @@ UniValue getnewaddress(const JSONRPCRequest& request)
         return NullUniValue;
     }
 
-    if (request.fHelp || request.params.size() > 1)
+    if (request.fHelp || request.params.size() > 2)
         throw std::runtime_error(
-            "getnewaddress ( \"account\" )\n"
+            "getnewaddress ( \"account\" ) ( \"style\" )\n"
```

Comment:
This shouldn't just be tacked on the end. Accounts are deprecated. Just replace "account" with an options Object.

Reply:
That would be API breaking for a lot of parties which is at odds with rapid deployment.

Reply:
https://github.com/bitcoin/bitcoin/pull/11403#discussion_r144199735

> This shouldn't just be tacked on the end. Accounts are deprecated. Just replace "account" with an options Object.

In this particular context, account will just be renamed to label, rather than removed entirely. (At least this is what #11536 does.)